charts/opentelemetry-collector/values.yaml

```diff
@@ -284,6 +284,8 @@ lifecycleHooks: {}
 #       - /test/sleep
 #       - "5"
 
+extraContainers: []
+
```

Comment:
Can you please move it closer to `initContainers`?

Reply:
Done.

Reply:
Done!